docs/astro/src/content/docs/reference/std-widgets/lineedit.mdx

```diff
@@ -140,3 +129,11 @@ LineEdit {
     }
 }
 ```
+
+### key-pressed(KeyEvent) -> EventResult
+Invoked when a key is pressed, the argument is a <Link type="KeyEvent" /> struct. The returned `EventResult` 
+indicates whether to accept or ignore the event. Ignored events are forwarded to the parent element.
```

Comment:
I think this is slightly inaccurate:

  - events are rejected instead of ignored
  - ignored events aren't forwarded to the parent element but handled by the `LineEdit` itself.

See below for a suggestion. This applies to `TextEdit` as well as `TextInput` itself, for both the `key-pressed` as well as the `key-released` callbacks.

```suggestion
Invoked when a key is pressed, the argument is a <Link type="KeyEvent" /> struct. Use this callback to
handle keys before `LineEdit` does. Return `accept` to indicate that you've handled the event, or return
`reject` to let `LineEdit` handle it.
```

Reply:
Please apply this fix also to `key-released` and then to `TextEdit`, and `TextInput`.